src/components/styles/book.css

```diff
@@ -1,11 +1,178 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
 .item {
-  border: 1px solid goldenrod;
-  width: 20rem;
-  padding: 1rem;
+  width: 80%;
+  padding: 2.5rem;
   margin-bottom: 1rem;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  border-radius: 0.5rem;
+  background-color: white;
+  margin-top: 1.5rem;
+}
+
+.display {
+  width: 15rem;
 }
 
 button {
   padding: 0.5rem;
   border-radius: 5px;
+  background-color: white;
+  border: none;
+  color: #4386bf;
+  font-family: "Roboto Slab", serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
```

Comment:
- [ ] I know it's been a while since we worked on a project with a pixel-perfect design, but we have a Zeplin design guideline for this one. Could you reproduce the template to achieve the project requirements? One of the objectives of this project is to understand how to apply styles to JSX as precisely as possible. ✍️

**Zeplin design**
![Screenshot from 2023-03-11 11-07-13](https://user-images.githubusercontent.com/17243325/224473257-12df092f-b230-4103-89c0-4eb5923d1518.png)

**Your design**
![Screenshot from 2023-03-11 11-07-33](https://user-images.githubusercontent.com/17243325/224473272-1e947d0b-43d0-4d03-9297-71f6a066d9a6.png)